modules/Bio/EnsEMBL/IO/Parser/BaseVCF4.pm

```diff
@@ -316,11 +316,10 @@ sub get_end {
     elsif(defined($info->{SVLEN})) {
       return unless $self->get_start;
       my $svlen = (split(',',$info->{SVLEN}))[0];
-      $end = $self->get_start + abs($svlen)-1;
+      $end = $self->get_start + abs($svlen) - 1;
     }
```

Comment:
```suggestion
    }
    elsif(($self->get_raw_info && $self->get_raw_info =~ /SVTYPE/) || ($alternatives && $alternatives =~ /\<|\[|\]|\>/)) {
      $end =  $self->get_start;
    }
```

Also, needs to define `$alternatives` at the start of the function -
```
my $alternatives = join(",", @{$self->get_alternatives});
```

**Explanation**:
The current changes in this PR effects two cases -
case 1: non SV VCF inputs (basically SNV and small indels where we have both REF and ALT strings defined using [ACTGN]).
case 2: SV cases (where ALT is symbolic) where `SVLEN` or `END` is not defined.

in case 2, with current changes we will get start > end, e.g. - 
```
2	68914092	del	T	<DEL>	.	.	.   
```
will return start = 68914093 and end = 68914092. The description of the VCF here is incomplete (how much is deleted), so it does not matter much what start and end in this case and they will get skipped ([see](https://github.com/Ensembl/ensembl-vep/blob/31a3581b84495b617b2f3980da6c6313ca6d238f/modules/Bio/EnsEMBL/VEP/Parser.pm#L582)). 

But it can matter in complex SV cases we support, such as breakends and tandem repeats, where in some cases SVLEN/END are not always needed -
```
2	68914092	BND00001121	A	]1:37938377]A	.	PASS	SVTYPE=BND
21	25587759	tr0	T	<CNV:TR>,<CNV:TR>	.	PASS	RUS=CAT,GT,CA;RUC=2,5,4;RN=2,1
```

So, we should consider cases where SV does not have SVLEN or END and make sure they are not getting skipped out.